### ISSUE TITLE
Server is reconfigured every time ansible runs

### ISSUE THREAD

**Issue by miguelaferreira** [softasap/sa-vpn-softether#6]
Every time I run Ansible the VPN server gets reconfigured, and that means new certificate.
New certificate means that all clients need to be reconfigured as well. Is there a way to stop this behaviour? And preferably, reconfigure the server if there were deviations in configuration, but not generate a new certificate?

**Comment by Voronenko** [comment]
@miguelaferreira Yes, for sure.

Quick question: what business scenarion causes you to re-run play each time?
Depending on answer I will try to address in a most efficient way

**Comment by miguelaferreira** [comment]
@Voronenko thanks for the quick reply.

I have a play that applies a few roles to a set of machines. It applies roles to manage packages, users, etc. Amongst those roles is `sa-vpn-softether` that sets up our VPN Server.

I need to run the play again if I need to change something on the machine that is not necessarily related to the VPN Server (e.g install a new OS package).
On the other hand, I also want to run the same play again when I need to add a new user to the VPN Server.

In either case, the play will run the `sa-vpn-softether` role and reconfigure my VPN Server.
It becomes a lot of work to reconfigure all the VPN clients every time.

**Comment by Voronenko** [comment]
Aha, I just apply in a different way
(A) Using ansible tags , i.e.
```
roles:
   - {  
      name: "runs on create / update",
      tags: 
         - create
         - update
      }
   - {  
      name: "runs on update",
      tags: 
         - update
      }
   - {  
      name: "runs always",
      tags: 
         - always
      }

then I just execute changes to boxes with tag update ;

or 
(B)
just split plays into different box bootstrap play, and application plays.

**Comment by miguelaferreira** [comment]
That makes sense. I've used Chef for a very long time and got used to the idempotent model. So I usually bake that into the Ansible roles I create. I guess I'll make a PR to your role, or embed your role in a role of mine. Thanks. I'll close this now.

**Comment by miguelaferreira** [comment]
One more question: do you have a construction to add new users to the VPN Server without reconfiguring it fully?

**Comment by Voronenko** [comment]
No at a moment , but this is question of introducing new scenario like  
`https://github.com/softasap/sa-vpn-softether/blob/master/templates/scenarios/create_hub`

but containing only part about adding new users.

**Comment by Voronenko** [comment]
Generally idempotency is issue here, thus let's leave Issue open.

**Comment by Voronenko** [comment]
Addressed in 1.1.1
